blocks/cards/cards.css

```diff
@@ -1,27 +1,91 @@
-.cards > ul {
-  list-style: none;
-  margin: 0;
-  padding: 0;
-  display: grid;
-  grid-template-columns: repeat(auto-fill, minmax(257px, 1fr));
-  grid-gap: 24px;
+.cards.suggested-videos {
+  margin: 20px 0;
 }
 
-.cards > ul > li {
-  border: 1px solid #dadada;
-  background-color: var(--background-color);
+.cards.suggested-videos h3 {
+  font-family: var(--body-font-family);
+  font-size: var(--body-font-size-m);
+  font-weight: var(--font-weight-medium);
+  color: inherit;
+  margin: 0 0 -2em;
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards.suggested-videos .card-wrapper{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 }
 
-.cards .cards-card-image {
-  line-height: 0;
+.cards.suggested-videos .card.block {
+  width: 100%;
+  border-top: 1px solid var(--light-grey);
+  padding: 25px 0;
 }
 
-.cards > ul > li img {
-  width: 100%;
-  aspect-ratio: 4 / 3;
+.cards.suggested-videos .card.block:last-of-type {
+  border-bottom: 1px solid var(--light-grey);
+}
+
+.cards.suggested-videos .card.block>div {
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
```

Comment:
Change this to:
```
display: flex;
align-items: center;
gap: 10px;
```

and with the other changes I am suggesting it should work out more centered and even.